recipes/rapidjson/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D RAPIDJSON_HAS_STDSTRING=ON ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D RAPIDJSON_BUILD_TESTS=OFF ^
+      -D RAPIDJSON_BUILD_EXAMPLES=OFF ^
+      -D RAPIDJSON_BUILD_DOC=OFF ^
+      -D CMAKE_VERBOSE_MAKEFILE=ON ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      ..
+
+if errorlevel 1 exit 1
+
```

Comment:
I don't know anything  about Windows, but I wonder if this extra line  influences in the `errorlevel`...